backend/api/v1/dashboards.py

```
@@ -1,53 +1,237 @@
 from fastapi import APIRouter, HTTPException, Depends
-from typing import List
+from typing import List, Dict, Any
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)